packages/components/molecules/f-searchbox/src/components/Form.vue

```diff
@@ -143,13 +146,13 @@ export default {
         /**
          * Display API suggestions in component: `form-search-suggestions`:
          *
-         * 1. Service layer should contain `autocomplete`.
+         * 1. Service layer should contain `autocomplete` or `isFullAddressSearchEnabled`.
```

Comment:
```suggestion
         * 1. Service layer should contain `isAutocompleteEnabled ` or `isFullAddressSearchEnabled`.
```

Reply:
This one is actually correct as it maps to an option called `autocomplete`. I have updated the comment so it's a little clearer.